src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -228,6 +228,11 @@ function EditorUI({ initialTextValue, dataSourceId }: EditorProps) {
     });
   }, [sendCurrentRequestToOpenSearch, openDocumentation]);
 
+  const tooltipDefaultMessage =
+    dataSourceId === undefined ? `Select a data source` : `Click to send request`;
```

Comment:
Do we need to check if data source plugin is enabled first?

Reply:
We don't need that check.

The dataSourceId passed in `src/plugins/dev_tools/public/application.tsx`, and it will be assure following scenario:

When MD is disabled:

initial dataSource id here is empty
When MD is enabled and hideLocalCluster = false

initial dataSource id here is empty
When MD is enabled and hideLocalCluster = true

initial dataSource id here is undefined
In all cases, once we choose a dataSource in dataSource picker, this dataSourceId will be the id get selected.